### DISCUSSION TITLE
Is it true? Sveltekit without Svelte?

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [sveltejs/kit#9307] [General]
I can't remember where I heard about it, but probably on some Youtube video.

Is it true that Sveltekit can be used with other frameworks and it's not locked to Svelte?

If so. Where can I read about it and find examples.

**Comment by vekunz** [comment]
If this is somehow possible, it is definitively not an official solution. But why would someone wants this? Without Svelte, SvelteKit would loose basically its complete sense. If you want Vue, use Nuxt. If you want React, use Next or Remix. If you want Angular, use Angular Universal.

**Reply by MrAmericanMike** [reply]
Right, I just though it was interesting that it was even possible and would be nice to see some examples.
